### ISSUE TITLE
When data is migrated in the EventMigrationHandler, we should automatically change stuff like dates.

### ISSUE THREAD

**Issue by halvors** [InfectedLan/InfectedAPI#166]
When data is migrated in the EventMigrationHandler, we should automatically change stuff like dates.'

One example case is the compos table where there is a starttime and endtime.

**Comment by petterroea** [comment]
Compos should technically not be migrated as they are lan-specific(even though we allways do the same :smile: .

**Comment by halvors** [comment]
Dates should be set relative to the event, so that we only have to change the event ID. See #195.

**Comment by petterroea** [comment]
Agreed. Do we have an api that converts to and from event-relative times?
